src/TogglePassword/doc/index.rst

```diff
@@ -0,0 +1,186 @@
+Symfony UX TogglePassword
+=========================
+
+Symfony UX TogglePassword is a Symfony bundle providing visibility toggle for password inputs
+in Symfony Forms. It is part of `the Symfony UX initiative`_.
+
+It allows visitors to switch the type of password field to text and vice versa.
+
+Installation
+------------
+
+Before you start, make sure you have `StimulusBundle configured in your app`_.
+
+Then, install this bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-toggle-password
+
+If you're using WebpackEncore, install your assets and restart Encore (not
+needed if you're using AssetMapper):
+
+.. code-block:: terminal
+
+    $ npm install --force
+    $ npm run watch
+
+    # or use yarn
+    $ yarn install --force
+    $ yarn watch
+
+Usage
+-----
+
+The most common usage of Symfony UX TogglePassword is to use it as a drop-in
+replacement of the native PasswordType class::
+
+    // ...
+    use Symfony\UX\TogglePassword\Form\TogglePasswordType;
+
+    class CredentialFormType extends AbstractType
+    {
+        public function buildForm(FormBuilderInterface $builder, array $options)
+        {
+            $builder
+                // ...
+                ->add('password', TogglePasswordType::class)
+                // ...
+            ;
+        }
```

Comment:
This part should be updated since you switched to a form extension :)

Reply:
Yes, i wanted to implement translation support first before updating doc 👍🏼 but i might do that first

Reply:
Documentation updated 👍🏼  (will do another update once translation support and button classlist customization will be updated 😄 )